MonoDevelop.MSBuild.Editor/Completion/MSBuildCompletionSource.cs

```diff
@@ -271,7 +231,7 @@ async Task<CompletionContext> GetPackageNameCompletions (MSBuildRootDocument doc
 				return CompletionContext.Empty;
 			}
 
-			var searchQuery = rr.Reference.ToString ().ToLower ();
+			var searchQuery = rr.Reference.ToString ();
```

Comment:
Minor nit, but you can cast to string here, no need to invoke the virtual method